Makefile

```diff
@@ -1,20 +1,24 @@
-.PHONY: create-cluster drop-cluster helm-install helm-uninstall helm-reinstall helm-deps-up
+.PHONY: create-cluster drop-cluster helm-install helm-uninstall helm-reinstall helm-deps-up validate-k-env
 
-NAMESPACE ?= tensorleap
+RELEASE ?= tensorleap
 
-cluster-create:
-	k3d cluster create ${NAMESPACE} --k3s-arg="--disable=traefik@server:0"
+validate-k-env:
+	[[ -x "$$(command -v kubectx)" && "$$(kubectx --current)" == 'k3d-tensorleap' ]]
+	[[ -x "$$(command -v kubens)" && "$$(kubens --current)" == 'tensorleap' ]]
 
-cluster-del:
-	k3d cluster delete ${NAMESPACE}
+cluster-create: validate-k-env
+	k3d cluster create ${RELEASE} --k3s-arg="--disable=traefik@server:0"
```

Comment:
1. It's actually not the "release" here, it's the cluster name
2. I think it should be `create tensorleap --config k3d-config.yaml` something like that